documentation/tutorial.md

```diff
@@ -68,3 +75,762 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
+```
+ubuntu@ip-172-31-11-104:~/data-integrator$ juju status
+Model     Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial  overlord    localhost/localhost  2.9.37   unsupported  10:32:09Z
+
+App                  Version  Status  Scale  Charm                Channel   Rev  Exposed  Message
+database-integrator           active      1  database-integrator              0  no       received mongodb credentials
+mongodb                       active      2  mongodb              dpe/edge   96  no
+
+Unit                    Workload  Agent  Machine  Public address  Ports      Message
+database-integrator/0*  active    idle   5        10.23.62.216               received mongodb credentials
+mongodb/0*              active    idle   0        10.23.62.156    27017/tcp
+mongodb/1               active    idle   1        10.23.62.55     27017/tcp  Replica set primary
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+1        started  10.23.62.55   juju-d35d30-1  focal       Running
+5        started  10.23.62.216  juju-d35d30-5  jammy       Running
+```
+To retrieve information such as the username, password, and database. Enter:
+```shell
+juju run-action  database-integrator/0 get-credentials --wait
+```
+This should output something like:
+```yaml
+​​unit-database-integrator-0:
+  UnitId: database-integrator/0
+  id: "24"
+  results:
+    mongodb:
+      database: test-database
+      endpoints: 10.23.62.55,10.23.62.156
+      password: VMnRws6BlojzDi5e1m2GVWOgJaoSs44d
+      replset: mongodb
+      uris: mongodb://relation-4:VMnRws6BlojzDi5e1m2GVWOgJaoSs44d@10.23.62.55,10.23.62.156/test-database?replicaSet=mongodb&authSource=admin
+      username: relation-4
+    ok: "True"
+  status: completed
+  timing:
+    completed: 2022-12-06 10:33:24 +0000 UTC
+    enqueued: 2022-12-06 10:33:20 +0000 UTC
+    started: 2022-12-06 10:33:24 +0000 UTC
+```
+Save the value listed under `uris:` *(Note: your hostnames, usernames, and passwords will likely be different.)*
+
+### Access the related database
+Notice that in the previous step when you typed `juju run-action  database-integrator/0 get-credentials --wait` the command not only outputted the username, password, and database, but also outputted the URI. This means you do not have to generate the URI yourself. To connect to this URI first ssh into `mongodb/0`:
+```shell
+juju ssh mongodb/0
+```
+Then access `mongosh` with the URI that you copied above:
+
+```shell
+mongosh "<uri copied from juju run-action  database-integrator/0 get-credentials --wait>"
+```
+*Note: be sure you wrap the URI in `"` with no trailing whitespace*
+
+You will now be in the mongo shell as the user created for this relation. When you relate two applications Charmed MongoDB automatically sets up a user and database for you. Enter `show dbs` into the MongoDB shell, this will output:
+```shell
+test-database  8.00 KiB
```

Comment:
This didn't show any output for me

![image](https://user-images.githubusercontent.com/116022293/211786175-76ccad18-d1d1-4429-94dd-e997ad8314d5.png)